src/screens/Components.js

```diff
@@ -284,6 +284,57 @@ export default class Components extends React.Component {
                 >
                   <LinearGradient colors={['transparent', 'rgba(0,0,0, 0.8)']} style={styles.cardGradient} />
                 </Card>
+                <Block width={330}>
+                  <Block flex style={{ padding: theme.SIZES.BASE }}>
+                    <Text h5>Sliders</Text>
+                  </Block>
+                  <Block flex style={styles.slider1}>
```

Comment:
You can now safely remove the Blocks encapsulating the Slider being the fact that the main Block has a `width` of `'100%'`.